theme-data/dark/dark-common.json

```diff
@@ -31,16 +30,35 @@
         "hover": "@color-yellow-20",
         "active": "@color-yellow-20"
       },
-      "highlight": "@theme-color-80",
+      "highlight": "@color-blue-80",
```

Comment:
we put 'theme' here because, right now, theme=blue. In the future, the design team said that they will have a theme-color for each theme.  Which is why we decided to call this token 'theme-color-', which, for now, redirect to 'color-blue-*', but in the future it will redirect to a different core color depending on the chosen theme.